src/main/java/org/tc/mail/MailNotificationSender.java

```diff
@@ -60,30 +67,30 @@ public void sendApprovalUpdate(Decision decision) {
             MimeMessageHelper helper = new MimeMessageHelper(message, true);
             helper.setSubject(COURSE_APPROVAL_UPDATE_SUBJECT);
             helper.setTo(decision.getCourseForReview().getOwner().getEmail());
-            String[] emailsCc = {roleService.getDepartmentManager().getEmail(),
-                    roleService.getKnowledgeManager().getEmail()};
+            String[] emailsCc = {roleService.getDepartmentManager().get().getEmail(),
+                    roleService.getKnowledgeManager().get().getEmail()};
             helper.setCc(emailsCc);
             helper.setText(contentBuilder.buildCourseApprovalUpdate(decision), true);
+            javaMailSender.send(message);
         } catch (MessagingException e) {
-            LOG.warn("Message didn't send on courseId {0}", decision.getCourseForReview().getId());
+            LOG.debug(String.format("Message didn't send on courseId %1$d", decision.getCourseForReview().getId()));
```

Comment:
If you use SLF4J you do not need String.gotrmat

log.warn("Failed to send message for course {}", courseId, ex);